kratos/input_output/cad_json_input.h

```diff
@@ -343,6 +348,21 @@ class CadJsonInput : public IO
         return std::make_tuple(outer_loops, inner_loops);
     }
 
+    static void ReadAndAddEmbeddedEdges(
+            typename BrepSurfaceType::Pointer pBrepSurface,
+            const Parameters rParameters,
+            typename NurbsSurfaceType::Pointer pNurbsSurface,
+            ModelPart& rModelPart,
+            SizeType EchoLevel = 0)
+    {
+        if (rParameters.Has("embedded_edges")) {

```

Comment:
One comment from my side. The error comes when the parameter has "embedded_edges" which is empty.

Reply:
true, thank you very much for that hint.